tools/get_tpm_pubhash/get_tpm_pubhash.go

```diff
@@ -17,33 +17,42 @@
 package main
 
 import (
-	"crypto/sha256"
-	"encoding/asn1"
-	"encoding/base64"
-	"flag"
 	"fmt"
 	"log"
 
 	"github.com/bloomberg/spire-tpm-plugin/pkg/common"
-)
-
-var (
-	tpmPath = flag.String("tpm-path", "/dev/tpmrm0", "location of tpm device")
+	"github.com/google/certificate-transparency-go/x509"
```

Comment:
wrong x509 package?

Reply:
Google mantains their own fork of `crypto/x509` that they use for TPMs. go-attestation uses this fork, so I have to as well.

Reply:
hmmph.